src/fides/api/ctl/database/seed.py

```diff
@@ -278,6 +288,44 @@ async def load_default_dsr_policies() -> None:
             except KeyOrNameAlreadyExists:
                 # This rule target already exists against the Policy
                 pass
+
+        consent_policy = Policy.create_or_update(
+            db=db_session,
+            data={
+                "name": "Default Consent Policy",
+                "key": DEFAULT_CONSENT_POLICY,
+                "execution_timeframe": 45,
+                "client_id": client_id,
+            },
+        )
+
+        consent_rule = Rule.create_or_update(
+            db=db_session,
+            data={
+                "action_type": ActionType.consent.value,
+                "name": "Default Consent Rule",
+                "key": DEFAULT_CONSENT_RULE,
+                "policy_id": consent_policy.id,
+                "client_id": client_id,
+            },
+        )
+
+        log.info("Creating: Data Category Access Rules...")
+        for use in ["advertising", "advertising.first_party", "improve"]:
+            try:
+                RuleUse.create(
+                    db=db_session,
+                    data={
+                        "key": use,
+                        "data_use": use,  # TODO: get rid of data_use, and have this be the key.
```

Comment:
Redundant pieces of data here, a RuleUse might just have a `key` which is the `use` like `advertising`, the `rule_id, and whether it's `executable`